lightning/src/ln/channelmanager.rs

```diff
@@ -4241,6 +4284,43 @@ where
 		Ok(())
 	}
 
+	fn unfunded_channel_count(
```

Comment:
I think it is the widely accepted order for readability, otherwise i am guessing "why do i need unfunded_channel_count" and it doesn't read like a story.

More details: https://softwareengineering.stackexchange.com/questions/199311/most-human-friendly-way-to-order-class-method-definitions

Reply:
how does moving `unfunded_channel_count` below `peers_without_funded_channels` change "on one screen" ?

These are not some arbitrary standards but generally accepted style guides.
[ Most humans read code tree in top down fashion, instead of reading unused functions and keeping them in-memory for later use :) ]